README.md

```diff
@@ -245,7 +245,7 @@ When you’ve run this example, you can expect a map that looks like this:
   ```bash
   gradle runExport \
     -PdataExportSpecFile='src/main/resources/executions/examples/london-cycle-traffic-air-quality-lsoa-backoff.json' \
-    -PoutputFile='/Desktop/london-cycle-traffic-air-quality-lsoa-backoff-output.json'
+    -PoutputFile='/Users/username/Desktop/london-cycle-traffic-air-quality-lsoa-backoff-output.json'
```

Comment:
Looks good. If you don't want the user to make any changes to it you could refer to home with `~` so it would be `~/Desktop/london-cycle-traffic-air-quality-lsoa-backoff-output.json`

Reply:
Forgot about that. Will implement.